frontend/src/features/upload/components/RecruitPeriodSelectInput.tsx

```
@@ -0,0 +1,103 @@
+            onClick={() => handleDropdown?.(!isOpen)}
+          >
+            <div className='font-medium'>{yearValue}</div>
+            {icon}
+          </button>
+
+          {hasDropdown && isOpen && (
+            <>
+              <div
+                className='fixed inset-0 z-10 cursor-default'
+                onClick={() => handleDropdown?.(false)}
+              />
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)